### PR TITLE
Gear page

### PR DESCRIPTION
**Pull request by RvanB**
A few adjustments were made to the entire website outside of the gear page, white backgrounds behind the main content area so text is more legible (instead of puzzle background), and some changes to containers to make PuzzlePageTitles work a little nicer.

### CODE REVIEW

File: client/components/public/Gear.jsx [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+  "btw": "XS-4XL",
+  "lstw": "S-3XL",
+  "lstm": "S-5XL",
+  "fcu": "S-5XL",
+  "hu": "S-5XL",
+  "hy": "S-XL",
+  "qzu": "S-3XL"
+}
+
+Gear = class Gear extends Component {
```
Comment (cabeese) [inline client/components/public/Gear.jsx:95]: Do you need the `Gear =` here? I think `class X extends Y` is sufficient

File: client/components/public/Gear.jsx [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+    this.setState({
+                  front: "/img/gear/" + str + "f.jpg",
+                  back: "/img/gear/" + str + "b.jpg",
+                  swatches: "/img/gear/swatches/" + str + ".png",
+                  size: sizes[str],
+                  title: titles[str],
+                  price: prices[str],
+                  materials: materials[str]});
+
+    this.setState({open: true});
```
Comment (cabeese) [inline client/components/public/Gear.jsx]: any reason this isn't part of the previous `setState`? doesn't matter much but this'll cause an extra re-render I think

File: client/components/public/Gear.jsx [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+  }
+
+  isTall() {
+    return window.innerWidth < window.innerHeight - 200;
+  }
+
+  getImageCard(code) {
+    title = titles[code];
+    path = "/img/gear/" + code + (forward[code] ? "f.jpg" : "b.jpg");
```
Comment (cabeese) [inline client/components/public/Gear.jsx]: These should both use "let" here (`let title = ...`, `let path = ...`)

File: client/components/team/Team.jsx [hunk 4]
```diff
@@ -35,14 +35,15 @@ Team = class Team extends Component {
...
     if (ready) {
-      content = team ? this._renderMain() : <Segment basic><NoTeamMessage><ProfileInvites user={user}/></NoTeamMessage></Segment>;
+      content = team ? this._renderMain() : <NoTeamMessage><ProfileInvites user={user}/></NoTeamMessage>;
     }
 
     return (
-      <Container>
+      <Segment basic>
         <PuzzlePageTitle title={this._getTitle()} subTitle={this.state.lastUpdate ? `Last Updated: ${this.state.lastUpdated}` : ''}/>
+        <Header as='h1' content="I'm here!"/>
```
Comment (cabeese) [inline client/components/team/Team.jsx]: Is this meant to be here?

**Review by cabeese — CHANGES_REQUESTED** [review]
I can't comment on the image files, so here's what I notice there:

* public/img/gear/flogo.png is HUGE - does it need to be?
* I see `buybutton.png` and `buybutton.gif` - are both needed?
* banner1.jpg says the THIRD GPH. I assume that's not used
* banner.jpg and TGPH-web-header-001.jpg appear to be the same image. Either one needed?
* is `gear/M 5 T-Color.png` used?
* `BuyNow-56a62a183df78cf7728ba82c.png` is very large

I saw you merged master into this branch... is that necessary? Was it to resolve some conflicts?

There is a commit that shrinks the images down. Can you "squash" that commit into the original commit that adds the image so that we don't have multiple copies of the image still in the repo history taking up space?

File: client/components/public/Gear.jsx (hunk 3 above)
Comment (RvanB) [reply client/components/public/Gear.jsx]: Fixed!

File: client/components/team/Team.jsx (hunk 4 above)
Comment (RvanB) [reply client/components/team/Team.jsx]: Removed!

File: client/components/public/Gear.jsx (hunk 2 above)
Comment (RvanB) [reply client/components/public/Gear.jsx]: Fixed!

File: client/components/public/Gear.jsx (hunk 1 above)
Comment (RvanB) [reply client/components/public/Gear.jsx:95]: You actually get a "Gear not defined" error from the routes if you don't include that! Tripped me up for a while.

**Pull request closed without merging**